src/modules/Dashboard/components/TakeNowModal/TakeNowModal.tsx

```diff
@@ -26,169 +21,350 @@ import { StyledImageContainer, StyledImg } from '../ActivitySummaryCard/Activity
-                value={subject}
-                options={participants}
-                onChange={setSubject}
-                data-testid={`${dataTestId}-take-now-modal-subject-dropdown`}
-                handleSearch={async (search) => {
-                  const response = await getWorkspaceRespondentsApi({
-                    params: {
-                      ownerId,
-                      appletId,
-                      search,
-                      limit: DEFAULT_ROWS_PER_PAGE,
-                    },
-                  });
-
-                  return (
-                    (response?.data as ParticipantsData)?.result.map(participantToOption) ?? []
-                  );
-                }}
-              />
-              <LabeledUserDropdown
-                label={t('takeNowModalParticipantLabel')}
-                name={'participant'}
-                tooltip={t('takeNowModalParticipantTooltip')}
-                placeholder={t('takeNowModalParticipantPlaceholder')}
-                value={participant}
-                options={participantsAndTeamMembers}
-                onChange={setParticipant}
-                data-testid={`${dataTestId}-take-now-modal-participant-dropdown`}
-                disabled
-              />
-            </Box>
+        <StyledFlexColumn sx={{ gap: 3.2, padding: theme.spacing(1.6, 3.2, 2.4) }}>
+          <StyledFlexTopCenter gap={2.4}>
+            <StyledImageContainer>
+              {!!activity.image && <StyledImg src={activity.image} alt={activity.name} />}
+            </StyledImageContainer>
+            <StyledHeadline sx={{ flexGrow: 1 }}>{activity.name}</StyledHeadline>
+          </StyledFlexTopCenter>
+          <StyledFlexColumn gap={2.4}>
+            <StyledFlexColumn gap={0.8}>
+              <StyledFlexColumn gap={0.4}>
+                <LabeledUserDropdown
+                  label={t('takeNow.modal.sourceSubjectLabel')}
+                  name={'participant'}
+                  placeholder={t('takeNow.modal.sourceSubjectPlaceholder')}
+                  value={sourceSubject}
+                  options={participantsAndTeamMembers}
+                  onChange={(option) => {
+                    setSourceSubject(option);
+                    if (option) {
+                      setIsSelfReporting(option.tag === 'Team');
+                    }
+                  }}
+                  data-testid={`${dataTestId}-take-now-modal-participant-dropdown`}
+                  handleSearch={(query) => handleSearch(query, ['team', 'participant'])}
+                  canShowWarningMessage={true}
+                  showGroups={true}
+                />
+                <FormControlLabel
+                  control={<Checkbox sx={{ margin: 0, width: 48, height: 48 }} />}
+                  sx={{ gap: 0.4 }}
+                  checked={isSelfReporting}
+                  onChange={(_e, checked) => setIsSelfReporting(checked)}
+                  disabled={true}
```

Comment:
That's true, according to the AC you should be able to do something like that. But I set it to always be disabled because I think it's not a valid use case. The only values available in the middle dropdown are team members.

I initially had it as `disabled={sourceSubject?.tag !== 'Team'}` which would allow this, but it felt like a weird use case. Do you think I should change it back?

<img width="1552" alt="image" src="https://github.com/ChildMindInstitute/mindlogger-admin/assets/14842108/50b6e19c-cffa-4343-a8c5-ab33e260d15f">

Reply:
After talking to @mbanting I do believe this would be a valid use case, especially for R2.

Example scenario:

* A group of users use a **single Admin account** for the Admin app.
* A member of this workspace sets up Take Now to be completed by **Manager A** using the Admin account.
* On the web app, the user logs in as **Manager A** and completes the assessment.

If we force the checkbox to always be enabled they would have to use the Admin account to complete the assessment